rfc/0023-kubewarden-stack-telemetry.md

```diff
@@ -0,0 +1,338 @@
+|              |                                  |
+| :----------- | :------------------------------- |
+| Feature Name | Kubewarden environment telemetry |
+| Start Date   | 2025-08-28                       |
+| Category     | telemetry                        |
+| RFC PR       | [fill this in after opening PR]  |
+| State        | **TO-BE-DEFINED**                |
+
+# Summary
+
+This RFC describes how the Kubewarden stack will send telemetry data about the
+environment in which it is running. The goal is not to collect information
+about user applications, but rather to learn more about the typical
+environments where Kubewarden is deployed. This data will help guide the
+project's future development.
+
```

Comment:
I would add a paragraph stressing out this can be turned off

Reply:
It depends if we want to opt-in or opt-out by default.
As a project maintainer I would prefer to opt-in by default, it provide more data
As a project user, I would prefer to opt out by default :)

Ultimately I am fine with both but we need to take a decision on this